extension/background/intentRunner.js

```diff
@@ -171,7 +171,7 @@ export class IntentContext {
         ) {
           return;
         }
-        const isGoogle = /^https:\/\/www.google.com\//.test(url);
+        const isGoogle = /\/search/.test(url);
```

Comment:
I think a somewhat more careful test here might be in order, but I might put it in real quick after landing this